etna/articles/models.py

```diff
@@ -120,6 +123,9 @@ class ArticlePage(
 
     new_label_end_date = datetime.now() - timedelta(days=21)
 
+    # DataLayerMixin overrides
+    gtm_content_group = "Stories from the archives"
```

Comment:
How about `gtm_content_group = HomePage.gtm_content_group` ?

Reply:
As revised in the requirements, `contentGroup1= "stories"` for all story pages i.e. Article index page, Article.